### PR TITLE
Update page_adoption-view.md

### PR DESCRIPTION
**Pull request by SeBuSov**
## WHAT

Test pullrequest

## FURTHER NOTES

Some text changes + comments -> can you see them?

### CODE REVIEW

File: docs/kit/adoption-view/page_adoption-view.md [hunk 1]
```diff
@@ -39,18 +40,20 @@ With the [EDC Connector][edc-url], a new communication component was introduced,
...
 - [Predictive Unit Real-Time Information Service (PURIS)][PURIS-url]
 - [Business Partner Data Management][BPDM-url]
-- [Digital Product pass][digital-product-pass-url]
+- [Digital Product Pass][digital-product-pass-url]
+<!-- Link to more Use Cases?-->
 
 ## Business Architecture
 
 The [EDC][edc-url] as a connector implements a framework agreement for sovereign, cross-organizational data exchange. The International Data Spaces Standard (IDS) and relevant principles in connection with [GAIA-X][gaiax-url] were implemented. The connector is designed to be extensible to support alternative protocols and to be integrated into different ecosystems.
+<!-- Redundant-->
```
Comment (SeBuSov) [inline docs/kit/adoption-view/page_adoption-view.md:56]: delete

File: docs/kit/adoption-view/page_adoption-view.md [hunk 2]
```diff
@@ -13,12 +13,13 @@ The ConnectorKit provides a connector framework, based on the [Eclipse Dataspace
 
 ### Vision
 
-We believe that sharing data along the automotive supply chain adds great value for every participant. The key to such collaboration is trust in the involved partners and assurance that the shared data is not being misused. Our vision is that in 5 years, companies along the automotive value chain will share and exchange data with each other as a matter of course, without anyone fearing that their data will be misused. This creates added value for everyone, which can only be achieved through collaboration.
+We believe that sharing data along the automotive supply chain adds great value for every participant. The key to such collaboration is trust in the involved partners and assurance that the shared data is not being misused. Our vision is that in 5 years, companies along the automotive value chain will share and exchange data with each other as a matter of course, without anyone fearing that their data will be misused. This unlocks added value for everyone as well as a significantly improved collaboration.
```
Comment (SeBuSov) [inline docs/kit/adoption-view/page_adoption-view.md:16]: Was redundant before. Shouldn't the vision also include a vision for the "Connector Kit"?

File: docs/kit/adoption-view/page_adoption-view.md [hunk 3]
```diff
@@ -13,12 +13,13 @@ The ConnectorKit provides a connector framework, based on the [Eclipse Dataspace
...
+<!-- Was redundant before. Shouldn't the vision also include a vision for the "Connector Kit"? -->
```
Comment (SeBuSov) [inline docs/kit/adoption-view/page_adoption-view.md:17]: Comparison to other KITs : I think we should more go in such a direction. Agree? I can formulate a proposal


Data Chain:
DataChainKit brings valuable data chain information to your use-cases and services through connected data that can help Business Owner and Catena-X participants to be up to date and prepared. It's easy to use the DataChainKit with an Open Source Software package, which can easily deployed via HELM or docker-compose. The DataChainKit enables to apply business logic along a distributed data chains, for example aggregation of certificates along the value chain. Also ad-hoc provisioning of continuous data chains across company boundaries for empowerment of use cases Circular Economy, Traceability, Quality and the European supply chain act.
https://eclipse-tractusx.github.io/docs-kits/kits/Data%20Chain%20Kit/Adoption%20View%20Data%20Chain%20Kit#vision

DT KIT https://eclipse-tractusx.github.io/docs-kits/kits/Digital%20Twin%20Kit/Adoption%20View%20Digital%20Twin%20Kit#vision--mission

**Review by jkbquabeck — APPROVED** [review]
unlocks

**Pull request closed without merging**